content/post/agile-analytics-p3.md

```diff
@@ -0,0 +1,91 @@
+---
+author: Michael Kaminsky
+title: "Agile Analytics, Part 3: The Adjustments"
+publishDate: "2018-07-01"
+draft: true
+tags: 
+ - agile
+ - scrum
+ - process
+---
+
+
+Agile software engineering practices have become the standard work management tool for modern software development teams. Are these techniques applicable to analytics, or is the nature of research prohibitively distinct from the nature of engineering?
+
+In this post I discuss some adjustments to the scrum methodology to make the process work better for analytics and data science teams.
+
+<!--more-->
+
+If you haven't already read them, you can find
+
+* [part 1 here]({{< relref "agile-analytics-p1.md" >}})
+* [part 2 here]({{< relref "agile-analytics-p1.md" >}})
+
+When managing a team of analysts and data scientists, we want to leverage [all of the best things]({{< relref "agile-analytics-p1.md" >}}) about the scrum methodology while adjusting for [the things that don't work so well]({{< relref "agile-analytics-p1.md" >}}). The general theme of this post is how to carve out some room in scrum for exploration, research, and iterative problem definition.
+
+# Key Adjustments to Scrum for Analytics
+
+  * Time-bound spikes for research
+  * Build in slack time for exploration
+  * Acceptance Criteria includes “write the next story” 
+  * Peer-review instead of sprint-review
+
+
+## Time-bound spikes for research
```

Comment:
In my experience, the difficulty with incorporating research into an agile framework is equating research points with non-research points. Since at the end of a sprint we usually tally up the total points done, we (as a team) must agree that a 5 point SPIKE investigating LTV is just as valuable as a 5 point ticket which produces tangible, stakeholder facing output. In other words, every point is made equal. I think with research it's harder to justify (but absolutely necessary).

Reply:
This is an interesting point that I want to avoid in this post for the sake of not getting too into the weeds on how to think about this. I think a separate post on the-value-of-research would probably be a better place to handle this. There's maybe a separate point that applying agile methodology to analytics work can confuse stakeholders about the purpose of an analytics team (i.e., they start treating them just like an engineering team) -- a really interesting point but also probably for a separate post :)